arduino/security/signature_test.go

```diff
@@ -22,14 +22,36 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestSignatureVerification(t *testing.T) {
-	res, signer, err := VerifyArduinoDetachedSignature(paths.New("testdata/package_index.json"), paths.New("testdata/package_index.json.sig"))
+var (
+	PackageIndexPath     = paths.New("testdata/package_index.json")
+	PackageSignaturePath = paths.New("testdata/package_index.json.sig")
+	BoardIndexPath       = paths.New("testdata/module_firmware_index.json")
+	BoardSignaturePath   = paths.New("testdata/module_firmware_index.json.sig")
+	BoardKey             = paths.New("keys/module_firmware_index_public.gpg.key")
```

Comment:
Move `keys/module_firmware_index_public.gpg.key` under `testdata` (we need it only for testing)